test/blueprints/Do.Test.Blueprints.Service.Test/Communication/MockCommunicationFeature.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Do.Communication;
+
+namespace Do.Test.Communication;
+
+public class MockCommunicationFeature : TestServiceSpec
+{
+    [Test]
+    public async Task Default_mock_behaviour_can_be_setup_for_given_client_type()
+    {
+        var client = MockMe.TheClient<Singleton>();
+
+        var response = await client.Send(new Request(UrlOrPath: string.Empty, Method: HttpMethod.Post));
+
+        response.ShouldNotBeNull();
+        ((string?)response.GetContentAsObject())?.ShouldBe("test result");
```

Comment:
leave a short comment here to lead a reader where to find this mock setup in the project